.github/workflows/valid-links.yml

```
@@ -4,16 +4,20 @@ on:
   push:
   pull_request:
   schedule:
-    - cron: '0 3 * * *'
+    - cron: "0 3 * * *"
 
 jobs:
   verify-links:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
       - run: |
-          git fetch --prune origin master
-          git checkout $GITHUB_SHA
+          git config --global user.email "action@github.com"
+          git config --global user.name "GitHub Action"
```

```suggestion
          git config --global user.name "github-actions[bot]"
```

To keep it consistent with the bot that is used when you use the Actions Token.